package.json

```diff
@@ -57,7 +57,9 @@
 			"@oclif/plugin-help"
 		],
 		"topics": {
-			"config": { "description": "Manages Lisk Commander configuration." }
+			"config": {
+				"description": "Manages Lisk Commander configuration."
+			}
```

Comment:
This change seems unrelated.

Reply:
prettify seems to have some random behavior.
anyone know why?

Reply:
@willclarktech @shuse2 I realized when you run npm install it formats package.json. I upgraded lisk-elements with npm command thus it introduced this change. I say we should keep this.

Reply:
Ah, with npm v5 it preserves indentation as well I think. OK!